src/main/scala/io/iohk/ethereum/consensus/pow/PoWConsensus.scala

```diff
@@ -65,15 +83,30 @@ class PoWConsensus private (
    *
    * TODO further refactors should focus on extracting two types - one with a miner, one without - based on the config
    */
-  private[this] def startMiningProcess(node: Node): Unit = {
-    if (minerRef.isEmpty) {
+  private[this] def startMiningProcess(node: Node, blocKCreator: PoWBlockCreator): Unit = {
```

Comment:
nitpick: `startMiningCoordinator`?

Reply:
Given that the mockedMiner is supposed to be removed, yes. Otherwise no. What do you think? :)